pyproject.toml

```diff
@@ -30,7 +30,7 @@ python = "^3.9, < 3.13"
 neo4j = "~5.20"
 neo4j-rust-ext = "^5.20.0.0"
 infrahub-sdk = {path = "python_sdk", extras = ["ctl","tests"], develop=true}
-pydantic = "~2.6"
+pydantic = "2.7.2"
```

Comment:
DO you want to hard pin to 2.7.2 or should we use `~2.7` instead?

Reply:
As we want to pin it to a specific version regardless even if we changed it to something else here it would still be pinned in the poetry.lock file and I think it makes more sense to have a good overview of which versions we have installed within this file.

Within the SDK we want the supported versions to be as broad as possible, where as the core should be more like "there can be only one" :)